### PR TITLE
I480 navigation styles

### PR DESCRIPTION
**Pull request by GaryRidgway**
Tracks #480

### CODE REVIEW

**Comment by bspeare** [comment]
Testing this.

File: src/components/menus/main-menu/main-menu--children.twig [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  "banner_size": 'large',
+  "banner_image": 'https://sandbox.prod.drupal.uiowa.edu/sites/sandbox.uiowa.edu/files/2021-03/pano.jpg',
+  "banner_vertical_alignment": 'banner--vertical-center',
+  "banner_horizontal_alignment": 'banner--horizontal-center',
+  "banner_title": '<span>Welcome</span> to Iowa',
+  "banner_text": '',
+  "banner_title_classes": 'headline--uppercase',
+  "banner_classes": '',
+  "banner_overlay": 'gradient-center',
+} %}
```
Comment (pyrello) [inline src/components/menus/main-menu/main-menu--children.twig:12]: Can we remove this to make the example as striped down as possible?

File: src/components/menus/main-menu/main-menu--children.twig [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{% if menu_type == 'main' %}
+<div class="menu-container {{ menu_bg }}">
+  <nav role="navigation" aria-labelledby="-menu" class="nav  main--secondary--horizontal block block-menu navigation menu--main">
```
Comment (pyrello) [inline src/components/menus/main-menu/main-menu--children.twig:3]: Can we remove any classes from this that are not UIDS-specific? This looks like it was output copied from a Drupal site.

Comment (GaryRidgway) [reply src/components/menus/main-menu/main-menu--children.twig:3]: yeah we can. I had just copied this over for testing purposes so we can strip it down.

**Comment by GaryRidgway** [comment]
The new updates look good Ben!

**Pull request merged**